client/src/client/database_manager_client.py

```diff
@@ -17,67 +17,174 @@
 class DatabaseManager:
     """Clientside database manager."""
 
-    def __init__(self, name: str, password: str) -> None:
+    def __init__(
+        self, name: str, password: str, username: str, color: str
```

Comment:
"color" argument does not really make sense in the constructor of a database manager. Another reason why "myself" should not be part of the DatabaseManager class

Reply:
Yeah, that makes sense.